### PR TITLE
Update ListSSHKeysForUser to match API definition allowing id or user…

### PR DESCRIPTION
**Pull request by pschwartz**
…name

Signed-off-by: Philip Marc Schwartz <Philip.Schwartz1@T-Mobile.com>

### CODE REVIEW

**Comment by pschwartz** [comment]
@svanharmelen Another fix correcting an API call to match the API doc.

File: users.go [hunk 1]
```diff
@@ -422,13 +422,17 @@ func (s *UsersService) ListSSHKeys(options ...RequestOptionFunc) ([]*SSHKey, *Re
 // https://docs.gitlab.com/ce/api/users.html#list-ssh-keys-for-user
 type ListSSHKeysForUserOptions ListOptions
 
-// ListSSHKeysForUser gets a list of a specified user's SSH keys. Available
-// only for admin
+// ListSSHKeysForUser gets a list of a specified user's SSH keys. A
```
Comment (svanharmelen) [inline users.go:425]:
```suggestion
// ListSSHKeysForUser gets a list of a specified user's SSH keys.
```

File: users.go [hunk 2]
```diff
@@ -422,13 +422,17 @@ func (s *UsersService) ListSSHKeys(options ...RequestOptionFunc) ([]*SSHKey, *Re
...
 // https://docs.gitlab.com/ce/api/users.html#list-ssh-keys-for-user
-func (s *UsersService) ListSSHKeysForUser(user int, opt *ListSSHKeysForUserOptions, options ...RequestOptionFunc) ([]*SSHKey, *Response, error) {
-	u := fmt.Sprintf("users/%d/keys", user)
+func (s *UsersService) ListSSHKeysForUser(uid interface{}, opt *ListSSHKeysForUserOptions, options ...RequestOptionFunc) ([]*SSHKey, *Response, error) {
+	user, err := parseID(uid)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	u := fmt.Sprintf("users/%s/keys", user)
```
Comment (svanharmelen) [inline users.go:435]:
```suggestion
	}
	u := fmt.Sprintf("users/%s/keys", user)
```

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Two very minor suggestions, looks fine otherwise!

**Review by svanharmelen — APPROVED** [review]
Thanks! LGTM 👍🏻

**Pull request merged**